examples/data/e2e_tests/setup_scripts/import_all.sh

```diff
@@ -45,10 +45,11 @@ fi
 import_function () {
     for var in `ls -d ${PWD}/${1}/*`; do
         canonicalPath=$(realpath "${var}")
+        fileContent=`cat ${canonicalPath}`
         curl -v \
             -H "@${HEADERS_FILE}" \
             -H 'Content-Type:application/json' \
-            --data "@${canonicalPath}" \
+            --data "${fileContent}" \
```

Comment:
What if the file content has some "powerful" characters like `"` or `\`?

Reply:
Tested:
- input #1: `"hdfsPublishPath": "/publish/uppercaseConform\\anceRule"`
- imported #1: `"hdfsPublishPath":"/publish/uppercaseConform\\anceRule"`
- input #2: `"outputDataType": "st\"ring"`
- imported #2: `"outputDataType":"st\"ring"`

Reply:
Cool, thank you. 😎